globals/card_definitions.gd

```diff
@@ -427,7 +429,7 @@ func get_effect_type_text(effect, card_name_source : String = "", char_effect_pa
 		"add_top_deck_to_gauge":
 			var topdeck_card = ""
 			if 'card_name' in effect:
-				topdeck_card = "(%s) " % effect['card_name']
+				topdeck_card = "([color=%s]%s[/color]) " % [CardHighlightColor, effect['card_name']]
```

Comment:
I really really want to be able to have this hover show the card, but I'm sure that's going to be a lot of work, haha.

Reply:
the color being hyperlink-blue probably doesn't help, admittedly